client/landing/stepper/utils/path.ts

```diff
@@ -57,8 +58,8 @@ export const useLoginUrl = ( {
 	redirectTo?: string | null;
 	pageTitle?: string | null;
 	loginPath?: string;
-} ): string => {
-	const locale = useLocale();
+	locale?: string;
+} ) => {
 	const localizedLoginPath = locale && locale !== 'en' ? `${ loginPath }${ locale }` : loginPath;
```

Comment:
Probably worth dropping a trailing slash around loginPath here.

 `import { trailingslashit } from 'calypso/lib/route';`

Alternatively I'd actually suggest just hardcoding login path here. It's something likely to be modified and not tested properly. e.g. someone abusing it to add query params but ignoring the concat for localization.

Reply:
Thank you @lsl, I added `trailingslashit`. 

We could probably hardcode the login path here, but I see the `onboarding-pm` flow using the `loginPath` parameter.

@chriskmnds, tagging you as I see you changed that code, I see that `onboarding-pm` [redirects to](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/onboarding-pm.ts#L141) `start/onboarding-pm` for logging in. The only reason I could find is to set the [footer link text](https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/signup-form/index.jsx#L1041). Would it be possible to use `start/account` and manage the text by passing the right `variationName` to it? Just to unify all the Stepper flows.
Just leaving this here, it's not super important now ✌️

Reply:
I am not sure. It sounds reasonable, but I don't know of any caveats off-hand cc @southp maybe?

Reply:
Thanks for the ping @escapemanuele , @chriskmnds .
`/setup/onboarding-pm` was deprecated per announcement made in p2-peP6yB-CQ. The `loginPath` parameter used to be more important than setting up the footer link text back then since we'd need it to decide on the correct redirects, but those have also been deprecated as I recall. So please feel free to remove it together with the `/setup/onboarding-pm` flow and bring better consistency to Stepper :)